modules/apps/collaboration/social/social-group-statistics-web/src/main/resources/META-INF/resources/add_activity_counter.jsp

```diff
@@ -23,6 +23,10 @@ String portletResourceNamespace = PortalUtil.getPortletNamespace(portletResource
 
 int index = ParamUtil.getInteger(request, "index");
 
+if (index == 0) {
+	index = 1;
+}
+
```

Comment:
Do we need this block?  When I was testing without this block of code, it looks like the fields increment correctly, only that it goes 0, 2,3,4...instead of 1,2,3,4.